views/metrics/components/chart-wrapper/index.tsx

```diff
@@ -10,9 +10,9 @@ const ChartWrapper: FC<PropsWithChildren<ChartHeaderProps>> = ({
 }) => (
   <Box
     width="100%"
+    height="100%"
     display="flex"
     borderRadius="xs"
-    overflowY="hidden"
     color="onSurface"
     gridColumn="1/-1"
```

Comment:
If direct parent is not display grid, please move it cuz there is not need for that. GridColumn hasn't effect for display flex.

Reply:
We give u more details about this on daily